pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/NamespacesTest.java

```diff
@@ -1249,9 +1249,9 @@ public void testSubscribeRate() throws Exception {
         Awaitility.await().untilAsserted(() -> assertFalse(consumer.isConnected()));
 
         // Out of limit period
-        Thread.sleep(6000L);
         pulsarClient.updateServiceUrl(lookupUrl.toString());
-        assertTrue(consumer.isConnected());
+        assertFalse(consumer.isConnected());
```

Comment:
This will be flaky. I'd rather have the previous revision since removing the Thread.sleep was fine.